### PR TITLE
Adds the ability to sign in to SNAC from SNAC Laravel using OAuth

### PR DESCRIPTION
**Pull request by rderik**
This commit includes the changes to Support Single Sign On between the
two apps. Changes had to be done on the SNAC application to support the
login and logout API calls.

### CODE REVIEW

**Comment by rderik** [comment]
WIP don't merge yet.

**Comment by rderik** [comment]
@glassjoseph I added the ability to login from SNAC to SNAC-Laravel, you can check the code. We should discuss it before merging it.

**Review by glassjoseph — APPROVED** [review]

**Pull request merged**
